src/lighten-or-darken.js

```diff
@@ -17,7 +17,6 @@ const lightenOrDarken = (col: string, amt: number): string => {
   let g: number = (num & 0x0000FF) + amt
 
   if (col[0] === '#') {
-    col = col.slice(1)
```

Comment:
this line wasn't used unless it was causing some side effect somewhere that the function was relying on

Reply:
This line does actually matter, but it's in the wrong place. I'll fix it after merging.